devicetypes/smartthings/zigbee-dimmer-power.src/zigbee-dimmer-power.groovy

```diff
@@ -130,6 +130,11 @@ def configure() {
         }
         // Level Control Cluster, command Move to Level, level start level, transition time 0
         cmds << zigbee.command(zigbee.LEVEL_CONTROL_CLUSTER, 0x00, sprintf("%02X0000", startLevel))
+    } else if (device.getDataValue("manufacturer") == "Jasco Products") {
+        // Some versions of hub firmware will incorrectly remove this binding causing manual control of switch to stop working
+        // These needs to be the first binding table entries because the device will automatically write these entries each time it restarts
+        cmds += ["zdo bind 0x${device.deviceNetworkId} 2 1 0x0006 {${device.zigbeeId}} {${device.zigbeeId}}", "delay 2000",
```

Comment:
is this different than

`zigbee.addBinding(___)`?

also tsk tsk for magic numbers

Reply:
`zigbee.addBinding` doesn't support changing the destination EUID